consensus/reactor.go

```diff
@@ -1402,9 +1402,9 @@ func (m *NewRoundStepMessage) ValidateBasic() error {
 	if !m.Step.IsValid() {
 		return errors.New("Invalid Step")
 	}
-	if m.SecondsSinceStartTime < 0 {
-		return errors.New("Negative SecondsSinceStartTime")
-	}
+
+	// NOTE: SecondsSinceStartTime may be negative
```

Comment:
really? huh

Reply:
It probably shouldn't be but I can't guarantee it easily